src/features/crops/components/Plants.tsx

```diff
@@ -33,8 +33,16 @@ export const Plants: React.FC = () => {
     gameService.send("item.sell", {
       item: selected.name,
       amount,
-    });
-    setToast({ content: "SFL +$" + selected.sellPrice.mul(amount).toString() });
+    })
+      
+    if (inventory["Green Thumb"]?.greaterThanOrEqualTo(1)) 
```

Comment:
why not use `getSellPrice()` from `pricing.ts` instead?

```js
setToast({ content: "SFL +$" + (getSellPrice(selected.name, inventory).mul(amount)).toString() });
```

Reply:
Spent alot of time trying to set it up with that, errors kept popping left and right so i came up with the alternative that I used here !